lms/envs/common.py

```diff
@@ -125,6 +125,12 @@
     # Can be turned off if course lists need to be hidden. Effects views and templates.
     'COURSES_ARE_BROWSABLE': True,
 
+    # Extra message to display in the Learner Dashboard when they are not enrolled in any courses yet.
+    'EMPTY_DASHBOARD_MESSAGE': None,
```

Comment:
Yep, that's clearer, thanks!  Addressed by https://github.com/edx/edx-platform/pull/15464/commits/c862059c609083ce196aff9a36ac6f657fc38b5d#diff-82bda515879c96ab0641bfcbe58631c3R129.